tests/test_glinet.py

```diff
@@ -81,6 +81,29 @@ async def test_connected_clients() -> None:
 	print(len(clients))
 	assert(len(clients) > 0)
 
+@pytest.mark.asyncio
+async def test_wifi_ifaces_get() -> None:
+	wifi_ifaces = await router.wifi_ifaces_get()
+	print(wifi_ifaces)
+	for iface in wifi_ifaces:

```

Comment:
This test fails as this just produces the string names.
```suggestion
	for iface in wifi_ifaces.values():
```